Functions/Import/xASL_imp_DCM2NII_Subject_StartConversion.m

```diff
@@ -1,4 +1,4 @@
-function [globalCounts, x, summary_line, destdir, scanpath, scan_name, dcm2niiCatchedErrors, nii_files, first_match] = xASL_imp_DCM2NII_Subject_StartConversion(globalCounts, x, bSkipThisOne, summary_line, destdir, scanpath, scan_name, dcm2niiCatchedErrors)
```

Comment:
scanfield has to be added to the parameter description and to the FORMAT.

Reply:
but of course a nice catch finding out that scanfield is necessary. 
This bugs goes for Michael who splitted functions without controlling the variables and content.

Reply:
This should now be fixed